website/src/css/lightbox_styles.module.css

```diff
@@ -0,0 +1,91 @@
+/* stylelint-disable docusaurus/copyright-header */
```

Comment:
What's this?

Reply:
I was getting 'could not resolve module...' warning while importing a stylesheet into the `lightbox.jsx`. This seemed to help, but now I see the issue must have been somewhere else. I'm removing this line.